src/github.ts

```diff
@@ -50,4 +52,35 @@ function EnhancedGitHubClient (options) {
   return octokit
 }
 
-module.exports = EnhancedGitHubClient
+export interface Options extends Octokit.Options {
```

Comment:
Not sure how important this is, but using `extends` here means that you need to specify these members _and everything that `Octokit.Options` expects_, which is why the build is currently failing.

I think `baseUrl` maps to `host` in Octokit, and `agent` is the other required member for `Octokit.Options` which comes from  [`http.Agent`](https://nodejs.org/dist/latest-v8.x/docs/api/http.html#http_class_http_agent).

Reply:
The `agent` was incorrectly marked as required but was fixed in https://github.com/octokit/rest.js/pull/803 (in `v15.2.3`).